### PR TITLE
feat: cleanup login example

### PR DESCRIPTION
**Pull request by ndom91**
ndom91 from NextAuth.js here!

First of all, thanks a lot for the example project and tutorial write-up!

When checking out the example, I ended up making a few changes and figured I'd open up a PR to upstream them if you're interested.

First, I cleaned up the environment variables. No longer is an explicit `DOMAIN` env var necessary. We can just grab the `host` off the already required `NEXTAUTH_URL`. Also `JWT_SECRET` isn't required explicitly any longer.

Also, I styled the 'Sign-in' button a bit and added a dummy 'fillmurray' user profile pic image just for fun.

For example:

image

### CODE REVIEW

**Review by w4ll3 — APPROVED** [review]

**Review by awoie — APPROVED** [review]
lgtm and thanks a lot!!

File: README.md [hunk 1]
```diff
@@ -5,18 +5,15 @@ to help developers get their hands on a fully functional and easy to understand
...
+## Getting Started
 
-Add the following to `.env.local.example` in order to run the application:
+Make a copy of `.env.local.example` to `.env.local`, and fill in the following two values. For example:
 
 ```
+NEXTAUTH_URL=http://localhost:3000
 NEXTAUTH_SECRET=somereallysecretsecret
-JWT_SECRET=itshouldbealsoverysecret
-DOMAIN=localhost:3000
```
Comment (awoie) [inline README.md:15]: if we remove those two lines from the config, the example won't run as expected. no sign out button will be displayed after login. @w4ll3 can you check what the reason for this is?

**Review by awoie — CHANGES_REQUESTED** [review]
I had some issues when I removed JWT_SECRET and DOMAIN from the config. The sign out button won't show. Can you check what the reason for this is @w4ll3 ?

File: README.md (hunk 1 above)
Comment (w4ll3) [reply README.md:15]: It works for me, couldn't reproduce on Linux nor Mac.

**Pull request merged**